android/runtime/v8/src/java/org/appcelerator/kroll/runtime/v8/ReferenceTable.java

```diff
@@ -134,4 +134,16 @@ public static Object getReference(long key)
 		}
 		return ref;
 	}
+
+	/**
+	 * Determines if the reference is strong
+	 *
+	 * @param key the key for the reference.
+	 * @return returns true if the reference is strong
+	 */
+	public static boolean isStrongReference(long key)
+	{
+		Object ref = getReference(key);
+		return !(ref instanceof WeakReference || ref instanceof SoftReference);
```

Comment:
what if getReference() returns null? This will return `true`, that it is a strong reference. Is that what we expect/want?

Reply:
I thought about that too 👍 

Yes, `null` returning `true` is okay here since we default to strong references and have validation for `null` references.